m4_sessions/1801-inning/group_6.md

```diff
@@ -17,3 +17,55 @@ Finding The Right Fit
 - Samuel Singer
 
 @emhickmann
+
+# Group Six
+
+### Pitch: 
+Are you one of those students who says you'll just take any job after Turing?? So were we! Come to our session to get a little more insight into your options, and how to narrow down your search to companies who are the right fit for YOU.
+
+### Session Title: 
+What's your favorite flavor of shit sandwich (and does it come with an olive?)
+
+https://markmanson.net/life-purpose
+
+
+### Session Outcomes:
+- Understand what drives you, where you do your best work, what makes you unique
+- Understand the pros and cons in the  with working at a start-up, mid-size, large company, agency vs. consultancy and how you fit within each
+
+### Session Outline:
+#### 10 mins- Introduction:
+1- 2 min self reflection:
+- what motivates you
```

Comment:
Might add, what motivates you to push yourself?  Or something that narrows this question down a bit more.